test/functional/fixtures/regression/gh-973/test.js

```diff
@@ -0,0 +1,26 @@
+describe('[Regression](GH-973) ', function () {
+    describe('Should leave extra distance between target and browser window while performing an action', function () {
+        it('Scroll to the upper left corner element in the document body', function () {
+            return runTests('./testcafe-fixtures/index.test.js', 'Scroll to upper left corner element');
+        });
+
+        it('Scroll to the lower right corner element in the document body', function () {
+            return runTests('./testcafe-fixtures/index.test.js', 'Scroll to lower right corner element', { skip: ['android', 'ipad', 'iphone'] });
+        });
+
+        // NOTE: we should use a different way to get the client area size on mobile devices
+        it('Scroll to the lower right corner element in the document body (mobile)', function () {
+            return runTests('./testcafe-fixtures/index.test.js', 'Scroll to lower right corner element (mobile)', { only: ['android', 'ipad', 'iphone'] });
+        });
+    });
+
+    describe('Should leave extra distance between target and parent container while performing an action', function () {
```

Comment:
describe('Should leave an extra distance between the target and parent container while performing an action', function () {